### PR TITLE
added cache header without gzip and subpath as file

### PR DESCRIPTION
**Pull request by ryadavso**
we can set just cache control header without compression(GZip)
we can apply do that all on file and folder wise as well

### CODE REVIEW

File: Options.cs [hunk 1]
```diff
@@ -54,6 +54,9 @@ class Options
             HelpText = "Subpath to replace files in (skip all other files).")]
         public string Subpath { get; set; }
 
+        [Option('z', "withoutgzip", Required = false, DefaultValue = false,
```
Comment (mderazon) [inline Options.cs]: I think it makes more sense to have the flag be opt-in and not opt-out.

I mean `gzip = true` makes more sense than `withoutgzip = false`

**Review by mderazon — CHANGES_REQUESTED** [review]
Please update the README as well

**Comment by ryadavso** [comment]
Changes done 
please review again

File: README.md [hunk 2]
```diff
@@ -63,7 +63,10 @@ Replacing .js files in-place using a connection string instead of host/key:
                             age header, in seconds.  Default 2592000 (30 days).
 
   -p, --subpath             Subpath to filter blob containter contents. All other
-                            files will be skipped.
+                            files will be skipped.File path can also be used here.
+
+  -z, --gzip                (Default: True) Apply Compression and CacheControl Header.
```
Comment (mderazon) [inline README.md:68]: can we split the gzip and cache control header into separate flags ?

**Pull request merged**